src/components/note/__internal__/status-with-tooltip/status.component.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledStatusIconWrapper from './status.style';
+import TooltipDecorator from '../../../../utils/decorators/tooltip-decorator/tooltip-decorator';
+
+class StatusIcon extends React.Component {
+  render() {
+    return (
+      <>
+        <StyledStatusIconWrapper
+          data-component='content-with-tooltip'
```

Comment:
Should it probably be like:
```suggestion
          data-component='status-with-tooltip'
```